.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,71 @@
+name: Pre-release
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  create-pre-release:
+    runs-on: ubuntu-latest
+    name: Build and push Docker image and create a new GitHub pre-release
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set environment variables
+        run: |
+          echo VERSION=$(cat VERSION).$GITHUB_RUN_NUMBER >> $GITHUB_ENV
+          echo BASE_IMAGE_NAME=$REGISTRY/$(echo ${GITHUB_REPOSITORY,,}) >> $GITHUB_ENV
+          echo COMMITED_AT=$(git show -s --format=%cI `git rev-parse HEAD`) >> $GITHUB_ENV
+
+      - name: Collect Docker image metadata (api)
+        id: meta-api
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.BASE_IMAGE_NAME }}-api
+          labels: |
+            org.opencontainers.image.created=${{ env.COMMITED_AT }}
+            org.opencontainers.image.version=v${{ env.VERSION }}
+            org.opencontainers.image.maintainer=Stijn Vermeeren <stijn.vermeeren@swisstopo.ch>
```

Comment:
Is it okay to provide your address here @stijnvermeeren-swisstopo  ? I could also use a generic contact address at SwissTopo

Reply:
Yes, this is fine.
However, I noticed that https://github.com/opencontainers/image-spec/blob/main/annotations.md lists a keyorg.opencontainers.image.authors but no key org.opencontainers.image.maintainer. Should we adjust this accordingly?

Reply:
I took it from the other GitHub project. However, you are right, I cannot find it in the documentation as well.